provider/cluster_rosa_classic_resource.go

```diff
@@ -1792,6 +1898,30 @@ func getThumbprint(oidcEndpointURL string, httpClient HttpClient) (thumbprint st
 	return sha1Hash(cert.Raw)
 }
 
+func (r *ClusterRosaClassicResource) getDefaultIngress(ctx context.Context, state *ClusterRosaClassicState) error {
+	ingresses, err := r.clusterCollection.Cluster(state.ID.Value).Ingresses().List().SendContext(ctx)
+	if err != nil {
+		return err
+	}
+	if ingresses.Size() > 0 {
```

Comment:
It is not possible to create a second ingress anymore, but if for some reason a customer imported or created a cluster using TF when it was still possible they might have more than one.

Might need to filter if the ingress being retrieved is actually the default one